### PR TITLE
Fix/drawer issues

### PR DESCRIPTION
**Pull request by chris-cedrone-cengage**
Fixing placement of drawer.

### CODE REVIEW

File: packages/react-magma-dom/src/components/TimePicker/TimePicker.tsx [hunk 1]
```diff
@@ -140,7 +143,10 @@ export const TimePicker = React.forwardRef<HTMLInputElement, TimePickerProps>(
             type="number"
             value={hour}
           />
-          <Divider> : </Divider>
+          <Divider isInverse={isInverse} theme={theme}>
```
Comment (silvalaura) [inline packages/react-magma-dom/src/components/TimePicker/TimePicker.tsx:146]: I think these changes snuck out in here, but are already a part of another PR. Should we revert?

Comment (greenervigil) [reply packages/react-magma-dom/src/components/TimePicker/TimePicker.tsx:146]: I do not see this on main.  I think we should keep this in.

**Review by greenervigil — APPROVED** [review]

File: packages/react-magma-dom/src/components/TimePicker/TimePicker.tsx (hunk 1 above)
Comment (silvalaura) [reply packages/react-magma-dom/src/components/TimePicker/TimePicker.tsx:146]: The changes are in this open PR: https://github.com/cengage/react-magma/pull/605/files

**Review by greenervigil — CHANGES_REQUESTED** [review]
@ccedrone The TimePicker is in another PR.  Should we remove it from this one?

File: packages/react-magma-dom/src/components/TimePicker/TimePicker.tsx (hunk 1 above)
Comment (ccedrone) [reply packages/react-magma-dom/src/components/TimePicker/TimePicker.tsx:146]: Oops, is there a fancy way to just remove the timepicker file?

**Review by greenervigil — APPROVED** [review]

**Pull request merged**